### PR TITLE
[MIRROR] Splits up 'Gross' food from 'Gore' and reshuffles taste preferences accordingly [MDB IGNORE]

### PR DESCRIPTION
**Pull request by SkyratBot**
Original PR: https://github.com/tgstation/tgstation/pull/68899
--------------------

## About The Pull Request

This PR goes through all of the food marked "GROSS" and replaces some of it with the tag ~~"CARRION"~~ "GORE".
This tag is supposed to represent "things which are visibly a corpse or an organ" while the GROSS tag now represents "things which are rotten or otherwise inedible".
The purpose of this change is largely that there is now a large amount of tiziran-specific food and it is limiting for it to share a taste category with acidic battery pie and urinal cakes. It's not intuitive for these things to be categorised in the same place as actual prepared food.

image

All kinds of butchered player meat are now Gore instead of exclusively butchered humans being Gross.
Zombie meat is now gross when it wasn't before, I don't know why it wasn't.
South Bronx Paradise bar (which was gross because it has unwanted insect larvae in it) is now Bugs instead of Gross, in case that's actually a plus for you.

Lizards now like Gore and dislike Gross.
Humans now dislike Gore.
Moths dislike Gore.
Flypeople like Gore as well as Gross.
Felinids now like Gore, but if there's a standing order that they're not allowed to like eating mice or some kind of convention on range of preferred food types I can easily take this out, it just seemed appropriate. 

The effects of this are that Lizards still like eating mice and Tiziran food but don't like eating rotten mice, burned mistakes, or baseballs. 
Deviant taste lizards don't like eating mice even if they are rotten.
Humans no longer automatically tolerate eating lizard organs but will enjoy it more than they did previously if they have deviant tastes (as well as human meat).
Deviant taste humans still enjoy mice, even if they are rotting.
Felinids like eating mice and quite a lot of Tiziran food.
Felinids are the only people who automatically like mice or organ burgers, because Lizards don't like the bread in burgers.
Moths aren't really affected much because most Gore is also Meat, but now get sad if they eat a butchered Podperson. A deviant tastes moth enjoys fruit and gore, an combination wihch I think describes blood tomatoes and nothing else.
Flypeople have Gore on their "liked" list because that covers most of what they'd previously eat. This isn't really a change for them.

This also has a side-effect where Tiziran food rots slightly slower, as that was apparently effected by having the GROSS tag.

**TL;DR** this isn't actually a very big change for anyone except felinids, most races have slightly fewer foods they will now enjoy as a result. Felinids make out pretty well out of this, if that becomes controversial I am extremely happy to snip this addition from them and have them dislike gore like humans do.
I just want the cats to eat mice ok.

## Why It's Good For The Game

The Gross food category was too expansive and covered too much of a competing range of concepts.
If you want to add some kind of joke food like a baseball burger or hardware cake which only someone with really weird preferences should enjoy it doesn't make intuitive sense that Lizards would also automatically enjoy them.
Conversely now that there's so much Lizard-centric cuisine it seems weird to simply label it all "gross" rather than being specific about why in a way which can affect future changes.

## Changelog

:cl:
refactor: A lot of food which was 'gross' is now 'gore'.
balance: 'Gore' is disliked by Humans and Moths, but liked by Lizards and Felinids. Lizards no longer like 'gross' food.
balance: All butchered player species are considered to be gore, only preferred by Lizards, Felinids, and people with Deviant Tastes. If you like the taste of your fellow crew, prepare accordingly.
/:cl:

### CODE REVIEW

**Comment by Elli-Skala** [comment]
This should be looked into for our custom species

**Comment by Tastyfish** [comment]
Just noticed this PR added GORE to both the like AND dislike list for teshari. Was it intended to just be disliked?

**Comment by tf-4** [comment]
> Just noticed this PR added GORE to both the like AND dislike list for teshari. Was it intended to just be disliked?

Oops. Yes.
